docs/_docs/frontmatter.md

```diff
@@ -65,9 +65,23 @@ front matter of a page or post.
 
           If set, this specifies the layout file to use. Use the layout file
           name without the file extension. Layout files must be placed in the
-          <code>_layouts</code> directory.
+          <code>_layouts</code> directory. 
 
         </p>
+        <ul>
```

Comment:
Note: `tbody td ul ` font-size is currently not the same as `tbody td p` elements.  This should be taken care of in another PR. (or just limit ourselves to `p` tags in tables.)